it/uk/gov/hmrc/personaldetailsvalidation/PersonalDetailsValidationISpec.scala

```diff
@@ -44,6 +54,27 @@ class PersonalDetailsValidationISpec extends BaseIntegrationSpec {
       verifyMatchingStatusInAuditEvent(matchingStatus = "success")
     }
 
+    "return OK and insert records into both databases when provided personal details can be matched by Authenticator" in new Setup {
```

Comment:
can we cover the three scenarios I mentioned please
can we make sure an IT test covers this scenario. i.e hit the route,
both db's inserted into - coverered
if association db not written to, submission still successful
if association db already has record in submission still succeeds and does not affect the user

Reply:
scenarios have now been covered

Reply:
still not addressed